etl/steps/data/meadow/oecd/2016-06-01/regional_wellbeing.py

```diff
@@ -0,0 +1,122 @@
+# # Regional wellbeing (OECD 2016)
+
+# +
+import pandas as pd
+
+from owid.catalog import Dataset, Table
+from owid.walden import Catalog as WaldenCatalog
+from etl.steps.data.converters import convert_walden_metadata
+
+# -
+
+# ## Load the data
+
+walden_ds = WaldenCatalog().find_one("oecd", "2016-06-01", "regional_wellbeing")
+df = pd.read_csv(walden_ds.local_path)
+
+# ## Drop useless columns
+
+# some columns are empty
+df.dropna(how="all", axis=1, inplace=True)
+
+# some columns have only a single value for the whole table -- not informative
+for col in df.columns:
+    if len(df[col].unique()) == 1 and col != "Time":
+        df.drop(col, axis=1, inplace=True)
+
+# ## Standardise names
+
+df.columns = [
+    "region_code",
+    "region_name",
+    "ind_code",
+    "ind_name",
+    "year",
+    "unit_code",
+    "unit_name",
+    "value",
+]
```

Comment:
I'd suggest using `df = df.rename(columns={'old_column': 'new_column', ...})` instead of renaming `df.columns = []`, which assumes that columns are in a certain order.

Reply:
Done